libs/core-ui/src/lib/util/JointDataset.ts

```diff
@@ -645,6 +645,34 @@ export class JointDataset {
     });
   }
 
+  public getJointDatasetFeatureName(userFeatureName: string | undefined): string | undefined {
+    // Return the joint dataset feature name for the given user feature name.
+    if (userFeatureName === undefined) {
+      return undefined;
+    }
+    let jointDatasetFeatureName = undefined;
+    let userDatasetFeatureName = undefined;
+    for (jointDatasetFeatureName in this.metaDict) {
+      if (
+        this.metaDict[jointDatasetFeatureName]
+          .abbridgedLabel === userFeatureName
+      ) {
+        userDatasetFeatureName =
+          this.metaDict[jointDatasetFeatureName]
```

Comment:
Can this be shorten by `userFeatureName` as `this.metaDict[jointDatasetFeatureName].abbridgedLabel === userFeatureName`

Reply:
Simplified the logic in the new iteration.